easybuild/easyconfigs/f/FlexiBLAS/FlexiBLAS-3.3.1-GCC-12.3.0.eb

```diff
@@ -38,7 +38,14 @@ components = [
         'source_urls':
         ['https://gitlab.mpi-magdeburg.mpg.de/api/v4/projects/386/packages/generic/flexiblas-source/v3.3.1/'],
         'sources': [SOURCELOWER_TAR_GZ],
-        'checksums': ['bbeebf5e5a006924558fec43f49affbe1aaa4cbacfc472a9ff6066ffda142e18'],
+        'patches': [
+            'FlexiBLAS-3.3.1_disable-lapack-tree-vectorize.patch',
+        ],
+        'checksums': [
+            'bbeebf5e5a006924558fec43f49affbe1aaa4cbacfc472a9ff6066ffda142e18',  # flexiblas-3.3.1.tar.gz
+            'd05e9d48255299428f3a8e2127bab3a3f80fa6de99340cfce34b6ab09f9b5884',
+            # FlexiBLAS-3.3.1_disable-lapack-tree-vectorize.patch
+        ],
         'backends': local_backends,
```

Comment:
based on slack discussion we can fix this using an arch specific block + `-mprefer-vector-width=256`

```
+if ARCH == 'x86_64':
+    components[0][2]['patches'] = 'FlexiBLAS-3.3.1_lapack-prefer-vector-width-256.patch'
+    components[0][2]['checksums'].append('deadbeef')
```

then change the patch to use `-mprefer-vector-width=256` instead of `-fno-tree-vectorize`